src/App.js

```diff
@@ -1,9 +1,10 @@
-
+import "./styles/styles.css";
+import { AppRoutes } from "./routes/AppRoutes";
 
 function App() {
   return (
     <div>
-    
+      <AppRoutes />
```

Comment:
I dont see the need to create another App.jsx when this App.jsx serves the same purpose. So, try to keep the code simple.